tests/all/main.rs

```diff
@@ -19,14 +19,9 @@ mod traps;
 mod use_after_drop;
 mod wast;
 
-// TODO(#1886): Cranelift only supports reference types on x64.
-#[cfg(target_arch = "x86_64")]
 mod funcref;
-#[cfg(target_arch = "x86_64")]
 mod gc;
```

Comment:
Tiny nitpick: these should be with the rest of the `mod` declarations above now (and sorted via `cargo fmt`).

Reply:
Done, thanks.